aws-lambda-java-runtime-interface-client/test/integration/codebuild/buildspec.os.alpine.yml

```diff
@@ -79,34 +79,7 @@ phases:
           --build-arg DISTRO_VERSION="${DISTRO_VERSION}"
   build:
     commands:
-      - set -x
-      - echo "Running Image ${IMAGE_TAG}"
-      - docker network create "${OS_DISTRIBUTION}-network"
-      - >
-        docker run \
-          --detach \
-          --name "${OS_DISTRIBUTION}-app" \
-          --network "${OS_DISTRIBUTION}-network" \
-          --entrypoint="" \
-          "${IMAGE_TAG}" \
-          sh -c "/usr/bin/${RIE} ${JAVA_BINARY_LOCATION} -jar ./HelloWorld-1.0.jar helloworld.App"
-      - sleep 2
-      - >
-        docker run \
-          --name "${OS_DISTRIBUTION}-tester" \
-          --env "TARGET=${OS_DISTRIBUTION}-app" \
-          --network "${OS_DISTRIBUTION}-network" \
-          --entrypoint="" \
-          "${IMAGE_TAG}" \
-          sh -c 'curl -X POST "http://${TARGET}:8080/2015-03-31/functions/function/invocations" -d "{}" --max-time 10'
-      - actual="$(docker logs --tail 1 "${OS_DISTRIBUTION}-tester" | xargs)"
-      - expected='success'
-      - |
-        echo "Response: ${actual}"
-        if [[ "$actual" != "$expected" ]]; then
-          echo "fail! runtime: $RUNTIME - expected output $expected - got $actual"
-          exit -1
-        fi
+      - aws-lambda-java-runtime-interface-client/test/integration/codebuild/scripts/run_invocation_test.sh
     finally:
       - |
         echo "---------Container Logs: ${OS_DISTRIBUTION}-app----------"
```

Comment:
nit: could also extract the cleanup logic to a separate script?

Reply:
I wanted to extract only in a single commit/PR what I "touch", but they logically belong together it's reasonable to extract them in the same PR, will do.

Reply:
Thanks for doing this!